test/pylib/scylla_cluster.py

```diff
@@ -78,7 +78,7 @@ def make_scylla_conf(workdir: pathlib.Path, host_addr: str, seed_addrs: List[str
         'skip_wait_for_gossip_to_settle': 0,
         'ring_delay_ms': 0,
         'num_tokens': 16,
-        'flush_schema_tables_after_modification': False,
+        'flush_schema_tables_after_modification': True,
```

Comment:
We don't want to do that. Schema commit log was introduced so we don't have to flush schema tables after modification.

Reply:
OK

> Fix: add a separate schema commit log for schema and use force-schema-commit-log

This is not good enough to make the test pass.

cc @kostja

Reply:
That's probably because you didn't use the correct syntax for enabling a config variable in conf/scylla.yaml:
```
'--force-schema-commit-log arg': 1,
```
the correct syntax is
```
'force_schema_commit_log': True
```

Reply:
Ugh, shouldn't the yaml file fail to load in this case?

Reply:
It wasn't this

'''
for option '--force-schema-commit-log' is invalid. Valid choices are 'on|off', 'yes|no', '1|0' and 'true|false')
'''

Reply:
Not `--force-schema-commit-log'`, but `'force_schema_commit_log': True`